README.md

```diff
@@ -204,6 +204,34 @@ Example:
         -DartifactId=test \
         -Dversion=1.0-SNAPSHOT
 
+### liberty-archetype-mp
```

Comment:
Can we also add a link to jump to this section at the top of the readme, similar to the other archetypes?

Reply:
Good catch, thanks.

Reply:
Done.